EIPS/eip-5283.md

```diff
@@ -0,0 +1,112 @@
+---
+eip: 5283
+title: Semaphore for Reentrancy Protection
+description: A Precompile-based parallelizable reentrancy protection using the call stack
+author: Sergio D. Lerner (@SergioDemianLerner)
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
```suggestion
type: Standards Track
category: ERC
```
```suggestion
type: Informational
```
This EIP is not a core EIP. Please pick one of these (I'm inclined towards the latter).

Reply:
Unless I'm reading this EIP wrong, it *is* a Core EIP as it introduces a precompile to the EVM.

Reply:
Disregard my previous comment.